docs/src/main/sphinx/release/release-405.md

```diff
@@ -0,0 +1,219 @@
+* Improve performance of reading Parquet files for `timestamp` and
+  `timestamp with timezone` data types. ({issue}`15204`)
+
+## Iceberg connector
+
+* Improve performance of reading Parquet files for `boolean`, `tinyint`,
+  `short`, `int`, `long`, `float`, `double`, `short decimal`, `UUID`, `time`,
+  `decimal`, `varchar`, and `char` data types. The
+  `parquet.optimized-reader.enabled` catalog configuration property can be set
+  to `false` to disable the optimized implementation. ({issue}`14423`, {issue}`14667`)
+* Improve performance when reading Parquet files. ({issue}`15257`)
+* Improve memory accounting for `INSERT`, `MERGE`, and `CREATE TABLE AS SELECT`
+  queries. ({issue}`14407`)
+* Improve performance of reading Parquet files for queries with filters. ({issue}`15268`)
+* Improve `DROP TABLE` performance for tables stored on AWS S3. ({issue}`13974`)
+* Improve performance of reading Parquet files for `timestamp` and
+  `timestamp with timezone` data types. ({issue}`15204`)
+* Fix `REFRESH MATERIALIZED VIEW` failure when the materialized view is based on
+  non-Iceberg tables. ({issue}`13131`)
+* Fix failure when an access denied exception happens while listing tables or
+  views in a Glue metastore. ({issue}`14971`)
+* Fix incorrect results when predicates over `row` columns on Parquet files are
+  pushed into the connector. ({issue}`15408`)
+
+## Kafka connector
+
+* Add support for [Protobuf encoding](https://developers.google.com/protocol-buffers/docs/encoding). ({issue}`14734`)
+
+## MongoDB connector
+
+* Add support for [fault-tolerant execution](/admin/fault-tolerant-execution). ({issue}`15062`)
+* Add support for setting a file path and password for the truststore and
+  keystore. ({issue}`15240`)
+* Add support for case-insensitive name-matching in the `query` table function. ({issue}`15329`)
+* Rename the `mongodb.ssl.enabled` configuration property to
+  `mongodb.tls.enabled`. ({issue}`15240`)
+* Upgrade minimum required MongoDB version to
+  [4.2](https://www.mongodb.com/docs/manual/release-notes/4.2/). ({issue}`15062`)
+* Delete a MongoDB field from collections when dropping a column.
+  Previously, the connector deleted only metadata. ({issue}`15226`)
+* Remove deprecated `mongodb.seeds` and `mongodb.credentials` configuration
+  properties. ({issue}`15263`)
+* Fix failure when an unauthorized exception happens while listing schemas or
+  tables. ({issue}`1398`)
+* Fix `NullPointerException` when a column name contains uppercase characters in
+  the `query` table function. ({issue}`15294`)
+* Fix potential incorrect results when the `objectid` function is used more than
+  once. ({issue}`15426`)
+
+## MySQL connector
+
+* Fix failure when the `query` table function contains a `WITH` clause. ({issue}`15332`)
+
+## PostgreSQL connector
+
+* Fix query failure when a `FULL JOIN` is pushed down. ({issue}`14841`)
+
+## Redshift connector
+
+* Add support for aggregation, join, and top N pushdown ({issue}`15365`)
```

Comment:
"top N" is not a user visible feature. It should be expressed in terms of `ORDER BY ... LIMIT`